Code/Mantid/Framework/PythonInterface/plugins/algorithms/DNSDetEffCorrVana.py

```diff
@@ -0,0 +1,244 @@
+# pylint: disable=no-init,invalid-name
+import mantid.simpleapi as api
+from mantid.api import PythonAlgorithm, AlgorithmFactory, MatrixWorkspaceProperty
+from mantid.kernel import Direction
+import numpy as np
+
+
+class DNSDetEffCorrVana(PythonAlgorithm):
+    """
+    Peforms detector efficiency correction on a given data workspace
+    using the Vanadium data.
+    This algorithm is written for the DNS @ MLZ,
+    but can be used for other instruments if needed.
+    """
+    properties_to_compare = ['deterota', 'wavelength', 'slit_i_left_blade_position',
+                             'slit_i_right_blade_position', 'slit_i_lower_blade_position',
+                             'slit_i_upper_blade_position', 'polarisation', 'flipper']
+    dataws = None
+    outws_name = None
+    vanaws = None
+    bkgws = None
+    vana_mean_name = None
+
+    def category(self):
+        """
+        Returns category
+        """
+        return 'PythonAlgorithms\\MLZ\\DNS\\CorrectionFunctions\\EfficiencyCorrections'
```

Comment:
@mganeva : the code looks very neat and ready to go. I'll be putting a few inline comments here.

I think that in the category string you have to use ';' as separator, whereas '\\' is for nested categories. So would this be 'PythonAlgorithms;MLZ\\DNS;CorrectionFunctions;EfficiencyCorrections'?